.github/workflows/release.yml

```
@@ -0,0 +1,26 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Release
+    timeout-minutes: 5
+    runs-on: [self-hosted, Linux, AWS]

```

```suggestion
    runs-on: ubuntu-latest
```
Since we're leaving this one public you'll want the GitHub runners